contracts/src/SemiFungible1155.sol

```diff
@@ -202,11 +223,11 @@ contract SemiFungible1155 is Upgradeable1155 {
             _valuesCache[0] = swapValue;
 
             for (uint256 i; i < len; ) {
-                _notMaxItem(currentID);
```

Comment:
This is still declared above, are we still using it?

Reply:
Nope, removed in latest commit. Thanks